content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'Get Started: Using and deploying models'
+description:
+  'Easily download your models from the model registry. Set up your CICD
+  pipelines to be trigger by model registry actions (such as assigning model
+  stages) and deploy models directly form the model registry.'
+---
+
+# Get Started: Using and Deploying Models
+
+In this chapter, we will learn how to access and use models and how to use the
+model registry to trigger automated CICD model workflows.
+
+## Pushing model artifacts to the remote
```

Comment:
I think it's worth mentioning that you will have to modify the existing remote or add a new one (and commit/push those changes to git) since there is an existing remote for this project that will not work.

Reply:
If we use the existing models on the previous page, it also saves us from making users push any new models here, which simplifies the workflow.

Reply:
Good point, let's roll with that. But then `dvc artifacts get` will have to refer not to the current repository but to our example repo. I think it is still better that way (and I can mention how you can use github env variables to refer to the current repo).